lib/simpleRequester.js

```diff
@@ -43,10 +43,10 @@ exports.do = options => {
         try {
           body = JSON.parse(body)
         } catch (err) {
-          reject(HttpErrors(res.statusCode, 'Invalid status code, expected 200'))
+          reject(new Error(`JSON parse error ${err}`))
           return
         }
-        reject(body)
+        reject(HttpErrors(res.statusCode, 'Invalid status code, expected 200'))
```

Comment:
Is there more information in body that has information that might pertain to what went wrong? Is so we should include that.